eventbus/kafka/eventbus.go

```diff
@@ -105,6 +114,14 @@ func NewEventBus(addressList, appID string, options ...Option) (*EventBus, error
 		Balancer:     &kafka.Hash{},    // Hash by aggregate ID.
 	}
 
+	if b.dialer != nil {
```

Comment:
Copy of 92-98?

Reply:
Ahh gotcha, could you add some comments above both transport configs saying it's for client... writer. So it's more clear?

Reply:
Just did 👍